partials/primary-nav.hbs

```diff
@@ -0,0 +1,35 @@
+<!-- partial:primary-nav -->
+{{#parseFixture 'default-content.json'}}
+    {{#with primary-nav}}
+
+        <nav id="js-primary-nav" class="primary-nav clearfix">
+            <ul class="primary-nav__list clearfix">
+            	{{#each this}}
+
+            		<li class="primary-nav__item{{#if children}} primary-nav__item--has-children{{else}} primary-nav__item--no-children{{/if}}">
```

Comment:
You might be able to just omit the "else" portion. If an item has no children that's usually the default behavior and thus the default styling.

Reply:
:thumbsup: